Backend/app.py

```diff
@@ -169,6 +199,24 @@ def public_class_list():
     status = 400
   return ResponseCreation.createResponse(res, status)
 
+@app.route('/admin/changerole/', methods=['POST', 'OPTIONS'])
+@auth.login_required
+def changeUserRole():
+  status = 200
+  if request.method =='OPTIONS':
+    return ResponseCreation.createEmptyResponse(status)
+
+  if g.authenticatedUser['userRole'] != "admin":
+    res = ResponseCreation.ControllerResponse()
+    res.addError('Role Error', 'Only admins can change user roles')
+    status = 403
+  else:
+    dynamoDBInstance = getDatabaseClient()
+    res = user_ctrl.changeUserRole(request.json, dynamoDBInstance)
+    if res.hasErrors():
+      status = 400
+  return ResponseCreation.createResponse(res,status)
```

Comment:
res is only defined inside the conditional branches and used outside of it. It's not an error and works fine but it just feels wrong.